crates/conjure_core/src/ast/domains.rs

```diff
@@ -193,6 +193,15 @@ impl Domain {
         }
     }
 
+    // turns vector of integers into a domain
+    pub fn make_int_domain_from_values_i32(&self, vector: &Vec<i32>) -> Option<Domain> {
```

Comment:
add a TODO note here to say this can be done more compactly in terms of the domain we produce.

for example instead of `int(1,2,3,4,5,8,9,10)` we can produce `int(1..5, 8..10)`